5_data_munge/src/munge_wqp_files.R

```diff
@@ -93,19 +93,123 @@ munge_wqp_withoutdepths <- function(in_ind, min_value, max_value, max_daily_rang
                                        f_to_c(ResultMeasureValue), ResultMeasureValue)) %>%
     mutate(`ResultMeasure/MeasureUnitCode` = 'deg C')
 
-  dat_daily <- group_by(dat_reduced, MonitoringLocationIdentifier, ActivityStartDate) %>%
-    summarize(temperature_mean_daily = mean(ResultMeasureValue),
-              temperature_min_daily = min(ResultMeasureValue),
-              temperature_max_daily = max(ResultMeasureValue),
-              n_obs = n(),
+  dat_reduced_statcode <- ungroup(dat_reduced) %>%
```

Comment:
What is the grouping structure for `dat_reduced`? You could consider moving the `ungroup()` step to line 95 to avoid repeating that step in line 96 and then again in line 109.